build/common.sh

```diff
@@ -100,20 +100,20 @@ kube::build::get_docker_wrapped_binaries() {
         );;
     "arm")
         local targets=(
-          cloud-controller-manager,armel/busybox
-          kube-apiserver,armel/busybox
-          kube-controller-manager,armel/busybox
-          kube-scheduler,armel/busybox
-          kube-aggregator,armel/busybox
+          cloud-controller-manager,arm32v7/busybox
```

Comment:
@ixdy `Alpine` is only available for `arm32v6`(for the first raspberry pi), but `arm32v6` binaries should be compatible with an `ARMv7` chip (which is likely the reason `Alpine` upstream has chosen to only support `arm32v6`).

If there is no specific requirements, I vote for using `arm32v7`(for Raspberry PI 2, Allwinner A10, A20, H3).